lib/types.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 
 const Gql = require('graphql-tag');
+const cloneDeep = require('lodash.clonedeep');
 
 const debug = require('debug')('graphql-component:types');
 
 const check = function (operation, fieldName, excludes) {
-  for (const [root, name] in excludes) {
+  for (const [root, name] of excludes) {
```

Comment:
here's the other issue I was mentioning. `root` was being destructured as `0` and `name` was `undefined`. it didn't matter much because this function would always return `false` and we're doing `!check(...)` below. I'm assuming the original intent of this function was to return `true` if the field should be excluded.

Reply:
doh...